### PR TITLE
Update windows-laps-overview.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes from Azure AD to Entra

Fixes https://github.com/MicrosoftDocs/memdocs/issues/4071

### CODE REVIEW

**Comment by msbemba** [comment]
> Intune

Thank you @Brenduns . I have made the change per your comment.

**Comment by Brenduns** [comment]
@msbemba  Thanks!

#sign-off

File: memdocs/intune/protect/windows-laps-overview.md [hunk 1]
```diff
@@ -87,9 +87,9 @@ Intune policy for Windows LAPS can configure a device to back up a local adminis
...
 
-    **Enable LAPS in Azure AD**:  
-    1. Sign in to the **Azure portal** as a [Cloud Device Administrator](/azure/active-directory/roles/permissions-reference#cloud-device-administrator).
-    1. Browse to **Azure Active Directory** > **Devices** > **Device settings**.
+    **Enable LAPS in Microsoft Entra**:  
+    1. Sign in to the **Microsoft Entra admin center** as a [Cloud Device Administrator](/azure/active-directory/roles/permissions-reference#cloud-device-administrator).
+    1. Browse to **Devices** > **Overview** > **Local administrator password recovery**.
     1. Select **Yes** for the *Enable Local Administrator Password Solution (LAPS)* setting and select **Save**. You may also use the Microsoft Graph API [Update deviceRegistrationPolicy](/graph/api/deviceregistrationpolicy-update?view=graph-rest-beta&preserve-view=true)
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/windows-laps-overview.md:93]:
```suggestion
    1. Select **Yes** for the *Enable Local Administrator Password Solution (LAPS)* setting and select **Save**. You may also use the Microsoft Graph API [Update deviceRegistrationPolicy](/graph/api/deviceregistrationpolicy-update?view=graph-rest-beta&preserve-view=true).
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**